pkgs/games/prismlauncher/wrapper.nix

```diff
@@ -32,21 +31,14 @@
 , textToSpeechSupport ? stdenv.isLinux
 , controllerSupport ? stdenv.isLinux
 
-  # Adds `glfw-wayland-minecraft` to `LD_LIBRARY_PATH`
-  # when launched on wayland, allowing for the game to be run natively.
-  # Make sure to enable "Use system installation of GLFW" in instance settings
-  # for this to take effect
-  #
-  # Warning: This build of glfw may be unstable, and the launcher
-  # itself can take slightly longer to start
 , withWaylandGLFW ? false
 
 , jdks ? [ jdk21 jdk17 jdk8 ]
 , additionalLibs ? [ ]
 , additionalPrograms ? [ ]
 }:
 
-assert lib.assertMsg (withWaylandGLFW -> stdenv.isLinux) "withWaylandGLFW is only available on Linux";
+assert lib.assertMsg (!withWaylandGLFW) "prismlauncher: withWaylandGLFW has no effect anymore.";
```

Comment:
might be nice to add a comment here about when it was added (similar to pkgs/top-level/aliases.nix) or a TODO so we can know (when) to drop this